### ISSUE TITLE
Metaforecast Prioritization

### ISSUE THREAD

**Issue by OAGr** [QURIresearch/potential-projects#9]
On Metaforecast, it could be neat to have prioritization for the following:
1. For each estimate, how important/decision-relevant is it?
2. How important is it that this variable be predicted better? (This could use (1), plus forecaster estimates)
3. How valuable has this question + its predictions been / expected to be?

I think most of the key questions now are on Metaculus, so maybe this could be done with Metaculus somehow.

**Comment by uvafan** [comment]
I'd be excited about QURI trying some version of this out. I think the MVP I'd be most excited about is manually curating important questions. For the first version with forecasts being made, I'd recommend sticking to one meta-question accompanying each question.

**Comment by OAGr** [comment]
Agreed. @NunoSempere did some early work on estimating the value of Metaculus questions, but there's still much more work to do. (Especially, to have the scores show up on Metaforecast and similar. Maybe they could be integrated into Metaculus one day).
